trimmer_decorator.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'decorator'
+class TrimmerDecorator < Decorator
+  def correct_name
+    name = @nameable.correct_name
+    return unless name.length > 10
+
+    name[0..9]
+  end
+end
```

Comment:
- [ ] Great catch! While testing the `correct_name` method in your trimmer decorator class, I noticed a small issue. When using characters greater than 10, it works perfectly fine. However, for characters less than 10, it returns `nil` instead of the original characters.

  To ensure the expected behavior, the correct_name method should return the original characters if the length is not greater than 10. I recommend addressing this issue to ensure consistent functionality. Keep up the good work!

Reply:
Thank @AbelG101, I see the errors, I corrected them and request the review again.